incuna-sass/_mixins.sass

```diff
@@ -1,4 +1,6 @@
+@import mixins/media
 @import mixins/retina
 @import mixins/svg
 @import mixins/triangle
 @import mixins/font-face
+@import mixins/clearfix
```

Comment:
we did decide not to reinvent ones that were already available by well used and supported libraries.